packages/cli/test/unit/util/deploy/create-git-meta.test.ts

```diff
@@ -125,4 +126,17 @@ describe('createGitMeta', () => {
       await fs.rename(join(directory, '.git'), join(directory, 'git'));
     }
   });
+  it('fails when `.git` is corrupt', async () => {
+    const directory = fixture('git-corrupt');
+    try {
+      await fs.rename(join(directory, 'git'), join(directory, '.git'));
+      await fs.copy(directory, join(os.tmpdir(), 'git-corrupt'));
+      const cpDirectory = join(os.tmpdir(), 'git-corrupt');
+
+      const data = await createGitMeta(cpDirectory, client.output);
+      expect(data).toBeUndefined();
+    } finally {
+      await fs.rename(join(directory, '.git'), join(directory, 'git'));
+    }
```

Comment:
Let's do the rename part in the tmp dir, and then clean up afterwards:

```suggestion
    const directory = fixture('git-corrupt');
    const tmpDir = join(os.tmpdir(), 'git-corrupt');
    try {
      // Copy the fixture into a temp dir so that we don't pick
      // up Git information from the `vercel/vercel` repo itself
      await fs.copy(directory, tmpDir);
      await fs.rename(join(tmpDir, 'git'), join(tmpDir, '.git'));

      const data = await createGitMeta(tmpDir, client.output);
      expect(data).toBeUndefined();
    } finally {
      await fs.remove(tmpDir);
    }
```

Reply:
This is much better, thanks